runtime/vm/jvminit.c

```diff
@@ -7822,6 +7825,9 @@ protectedInitializeJavaVM(J9PortLibrary* portLibrary, void * userData)
 		}
 	}
 #endif /* defined(J9VM_OPT_JFR) */
+#if defined(OMR_THR_YIELD_ALG)
+	omrthread_monitor_init_with_name(&vm->cpuUtilCacheMutex, 0, "CPU Utilization Cache Mutex");
```

Comment:
Maybe move the create/destroy of the monitor to `vmthinit.c` where many other global monitors are managed.

Reply:
Dozens of monitors are initialized in `initializeVMThreading` and destroyed in `initializeVMThreading`

Reply:
Sorry, read the file name wrong, updated.